test/support/ae_mdw/db/mutation_case.ex

```diff
@@ -0,0 +1,24 @@
+defmodule AeMdw.Db.MutationCase do
+  @moduledoc """
+  Test case setup providing an empty memory store.
+  """
+
+  use ExUnit.CaseTemplate
+
+  using do
+    quote do
+      import AeMdw.TestUtil
+    end
+  end
+
+  setup tags do
+    alias AeMdw.Db.MemStore
+    alias AeMdw.Db.NullStore
+
+    if Map.get(tags, :integration, false) or Map.get(tags, :skip_store, false) do
```

Comment:
I would remove adding any conditionals to this, or adding new configuration values.

We're simply creating a new store object which won't affect the way records are retrieved by default

Without this tests that have this pattern match `test "test", %{store: store} do` are simply going to fail

Reply:
This is needed to allow a single setup for all test cases by `setup_all` like here:
https://github.com/aeternity/ae_mdw/blob/8fb32efa103cdf135d9c0c4df1aee976fee4786e/test/ae_mdw_web/controllers/aexn_transfer_controller_test.exs#L32